src/reconcile/rec_write.c

```diff
@@ -2870,9 +2877,12 @@ __rec_split_write(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WTI_REC_CHUNK *chu
              * Discard the delta if it is larger than the configured percentage of the size of the
              * full image.
              */
-            if (build_delta &&
-              ((r->delta.size * 100) / chunk->image.size) > conn->page_delta.delta_pct)
-                build_delta = false;
+            if (build_delta) {
+                header = (WT_PAGE_HEADER *)r->delta.data;
```

Comment:
Looks like we can define this outside the `if` line 2870, so we don't have to do it here and repeat it 2891

Reply:
no, that is not correct. Header is only available after __rec_build_delta